### DISCUSSION TITLE
questions of ANI & True positive

### DISCUSSION THREAD

**Discussion by wen1112** [bhattlab/phanta#21] [General]
Hi, I want to know when calculating statistics related to true and false-positive species, phanta are using ANI ≥ 95% & coverage ≥ 85%, or just ANI≥ 95%. And I saw fastANI was used, but "anicalc.py script provided by checkV" also metioned, when make ture the true and false-positive species, which one with tools result hav been used? Because when non-db you metion the checkv, but when infant data you metion fastANI only, so I'm not sure which tool's results I should be using.
In addition， when you calculating statistics related to true and false-positive species in non-db or MGV-Refseq related, ANI ≥ 95% only it can only mean that there is a high sequence similarity between them, do their true taxonomic profiles need to be checked? Could there be inconsistencies?

**Comment by meenachakra** [comment]
Hi Wen112, Thank you for your interest in Phanta. Could you please provide more detail on your application? That will help with answering your questions. True and false positive species would not apply in the infant data case (since those are not simulations), so want to make sure we understand what you mean by this.

**Comment by wen1112** [comment]
For example, my simulations data are from GPD and GVD, but the phanta result is OTU from MGV and RefSeq species name. I know the OTU has a representative sequence from MGV. So I want to know how can I make sure the True or false positive? If I should calculate just ANI >=95 using fastANI or something else? Thank you!

**Comment by wen1112** [comment]
oh, I have another question to add as. When I find a relate with Phanta and GPD DB, I calaluate fastANI just as MGV-RefSeq(--fragLen 500; ANI>=95), the coverage should be considered? The Phanta was used default or not?

```
--minFraction <value>
     minimum fraction of genome that must be shared for trusting ANI. If
     reference and query genome size differ, smaller one among the two is
     considered. [default : 0.2]
```

**Comment by meenachakra** [comment]
Thank you for these details - will discuss with @yipinto; we will get back to you within a couple days.

**Comment by yipinto** [comment]
Hi @wen1112,
Thanks for you questions!
First, we expect to have a database which will include GPD and GVD soon!
You may use any tool to calculate average nucleotide identity, fastANI should be much faster and good if you want to have many pairwise comparisons. We used ANIcalc (and not fastANI) to compare MGV and GPD genomes. 
As for how to define what is FP classification, you can consider several approaches, closest genome or species/above ANI threshold/both of these. 
Another option you can consider is to cluster GPD genome with MGV genomes, then you will know the "right" MGV OTU per GPD genome.
Hope it answered your questions.

**Reply by wen1112** [reply]
My questions have been explained, thank you for your reply！
